src/sdk/DialogServiceConnector.ts

```diff
@@ -144,6 +144,10 @@ export class DialogServiceConnector extends Recognizer {
         return this.privProperties;
     }
 
+    public set activityTemplate(messagePayload: string) {
```

Comment:
So this is a property on the connector not on the config?

Reply:
Correct, this enables the client to "stamp" their activities and have that persist on following turns, until they set/reset it. Though on the topic of properties, maybe this better belongs in our property collection (per conversation w/Ryan, below)